doc/handler.md

```diff
@@ -0,0 +1,52 @@
+### Handler
+
+A handler, is responsible for the execution of a query / command. it can in the case of a query return the requested information with the criteria present in the body of the query and in the case of a command simply do what he asked him
+
+#### How to register handler
+
+```yaml
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    public: false
+
+  _instanceof:
+    Nihilus\CQRSBundle\Query\QueryHandlerInterface:
+      tags:
+        - { name: 'nihilus.cqrs.query_handler' }
+    Nihilus\CQRSBundle\Command\CommandHandlerInterface:
+      tags:
+        - { name: 'nihilus.cqrs.command_handler' }
```

Comment:
Why not use the extension `NihilusCQRSExtension` and add:
```php
$container->registerFo
rAutoconfiguration(\Nihilus\CQRSBundle\Query\QueryHandlerInterface::class)
    ->addTag('nihilus.cqrs.query_handler')
;
$container->registerForAutoconfiguration(\Nihilus\CQRSBundle\Command\CommandHandlerInterface::class)
    ->addTag('nihilus.cqrs.command_handler')
;
```

Because `_instanceof` works only for the services define in the same file, so we will have to put it in multiple places.

Reply:
J'avais oublié cette feature du container, merci je vais faire cela

Reply:
fait